Dockerfile

```diff
@@ -1,6 +1,4 @@
-ARG PHP_VERSION=8.1.8
-
-FROM php:${PHP_VERSION}-fpm-alpine
+FROM php:8.1.8-fpm-alpine
```

Comment:
Lets back to ARG version.
Also maybe lets change version to:
```Dockerfile
ARG PHP_VERSION=8.1.8-fpm-alpine
```
It will allow us to test different images in GA.
E.g. `8.2-rc-fpm-alpine` or `latest`

Reply:
changed.